canvas_modules/common-canvas/src/object-model/object-model.js

```diff
@@ -947,6 +947,13 @@ export default class ObjectModel {
 		return notificationMessages;
 	}
 
+	deleteNotificationMessages(ids) {
+		// accept a single key or array of keys
```

Comment:
This is a great example of an unnecessary comment which has already not managed to keep up with changes to the code -- which is often what happens with comments in the code. :-)  I would remove it but if you want to keep it it should be corrected.

Reply:
removed, I agree this is unnecessary